packages/EasyActivity/tests/Stubs/CircularReferenceHandlerStub.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyActivity\Tests\Stubs;
+
+use EonX\EasyActivity\Bridge\Symfony\Serializers\CircularReferenceHandlerInterface;
+
+final class CircularReferenceHandlerStub implements CircularReferenceHandlerInterface
```

Comment:
Why do we use a Stub instead of a real service?

Reply:
done